config/swagger.yaml

```diff
@@ -2638,9 +2638,6 @@ definitions:
       state_hash:
         $ref: '#/definitions/EncodedHash'
         description: 'Channel''s next state_hash'
-      block_hash:
```

Comment:
As this was re feature introduced for Fortuna though never used, this shall not impact users. Though I suggest adding release note line mentioning removal of this.

Reply:
Still suggested though not strictly needed.